keygen/src/keygen.rs

```diff
@@ -72,11 +73,17 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                         .long("force")
                         .help("Overwrite the output file if it exists"),
                 )
+                .arg(
+                    Arg::with_name("no_passphrase")
+                        .short("n")
```

Comment:
Actually... I changed my mind 🙃 

Let's drop the short opt here.  Make it a little painful to do it wrong

Reply:
"wrong" 😝

Reply:
👍 shortopts are the worst